lists/1313161554/list.json

```
@@ -777,6 +777,14 @@
       "decimals": 18,
       "chainId": 1313161554,
       "logoURI": "https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/0xa69d9Ba086D41425f35988613c156Db9a88a1A96/logo.svg"
-    }
+    },
+    {
+      "name": "BNB - Allbridge",
+      "address": "0xb14674c7264ec7d948b904aab2c0e0f906f6e762",
+      "symbol": "abBNB",
+      "decimals": 18,
+      "chainId": 1313161554,
+      "logoURI": "https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/0xb14674c7264ec7d948b904aab2c0e0f906f6e762/logo.svg"
+    },
```

This is incompatible json format
```suggestion
    }
```